docs/Hosts/Spawn-Hosts.md

```diff
@@ -50,14 +50,34 @@ Clicking it will pre-populate the spawn host page with a request to spawn a host
 
 ![spawn_host_modal.png](../images/spawn_host_modal.png)
 
-Fetching artifacts can also be performed manually; see [fetch](../CLI#fetch) in the Evergreen command line tool documentation.
+The spawn host will not fetch the binaries and artifacts associated with the task on start-up because it lacks the necessary permissions. After you SSH into the host, you will need to run the following command to fetch them (which will prompt you to authenticate):
+
+```sh
+evergreen host fetch
+```
+
+Note: You will need to run this command for each spawn host you create that fetches the artifacts of a task.
+
+If you do not want the pre-made fetching options, you can use the Evergreen CLI command [fetch](../CLI#fetch).
```

Comment:
I think we should elaborate on what pre-made fetching options means here for users who would be missing some of the context.

Reply:
I added a suggestion below that you can use if you want to
Edited to add: nvm I was looking at the old file.